control-base/src/main/java/fi/nls/oskari/control/data/GetGeoPointDataHandler.java

```diff
@@ -82,8 +82,12 @@ public void handleAction(final ActionParameters params) throws ActionException {
 
 			final OskariLayer layer = mapLayerService.find(layerId);
 			final String layerType = layer.getType();
-
-			if (OskariLayer.TYPE_WMS.equals(layerType)) {
+			//HSY doesn't want GFI from basemaps.
+			boolean isBaseMap = false;
```

Comment:
Couldn't resist doing it for you: https://github.com/oskariorg/oskari-server/commit/3ffdac1c9f40fa0e1d29e9b122340c20f3931782